bin/journey-test.sh

```diff
@@ -215,12 +233,12 @@ solve_all_exercises() {
 
     pushd ${exercism_exercises_dir}/${TRACK}/${exercise}
     # Check that tests compile before we strip @Ignore annotations
-    gradle compileTestJava
+    "$EXECPATH"/gradlew compileTestKotlin
     # Ensure we run all the tests (as delivered, all but the first is @Ignore'd)
     for testfile in `find . -name "*Test.${TRACK_SRC_EXT}"`; do
-      sed 's/@Ignore//' ${testfile} > "${tempfile}" && mv "${tempfile}" "${testfile}"
+      sed 's/@Ignore\(\(.*\)\)\{0,1\}//' ${testfile} > "${tempfile}" && mv "${tempfile}" "${testfile}"
```

Comment:
Ah, that makes sense - I thought it was strange because I thought [this](https://github.com/exercism/xkotlin/blob/master/exercises/build.gradle#L51) would take care of it, but that is only if you run `gradlew test` on the root which then runs all the tests.